projects/hslayers/src/components/layermanager/layermanager.service.ts

```diff
@@ -237,10 +237,11 @@ export class HsLayerManagerService {
       layerDescriptor.thumbnail = this.getImage(layer);
       this.data.baselayers.push(<HsBaseLayerDescriptor>layerDescriptor);
     }
+    //*NOTE Commented out, because the  following references to this.data.baselayer are causing issues.
```

Comment:
Issues of what kind?

Reply:
@raitisbe Incorrectly sets base layer visibility for compositions on mapWhiteboard. For example, when opening composition with base layer that is other then OpenStreetMap, at first everything works fine. When (selecting from My Compositions list) the same composition is selected, none of base layers are visible. After some changes made on mapWB side, everything is working fine. Didn't find any use cases other than in Layermanager service, that is why I commented it out, for future discussions.